bindings_generator/src/documentation.rs

```diff
@@ -52,7 +52,8 @@ pub fn generate_class_documentation(api: &Api, class: &GodotClass) -> TokenStrea
     let official_docs = format!(
         r#"## Official documentation
 
-See the [documentation of this class]({url}) in the Godot engine's official documentation."#,
+See the [documentation of this class]({url}) in the Godot engine's official documentation.
+The method descriptions are generated from it and typically contain code samples from GDScript, not Rust."#,
```

Comment:
Nitpick: *in* GDScript or *for* GDScript?

Reply:
Good remark, fixed 🙂